tests/eth-tests.js

```diff
@@ -20,9 +21,21 @@ const rawUnsignedTransaction = (chainId, unsignedTxParams) => {
         bnToRlp(new BN(chainId)),
         Buffer.from([]),
         Buffer.from([]),
-    ]);
+    ]);   //  return unsignedTx.getMessageToSign(false);
+
+};
+
+const rawUnsignedEIP1559Transaction = (chainId, unsignedTxParams) => {
+  const common = Common.forCustomChain(1, { name: 'avalanche', networkId: 1, chainId }, 'london');
+
+  const unsignedTx = EIP1559Transaction.fromTxData({...unsignedTxParams}, { common });
+ 
+
+  // https://github.com/ethereumjs/ethereumjs-monorepo/issues/1188
```

Comment:
Oh is this fixed now? Let's use it in `rawUnsignedLegacyTransaction` to keep consistency

Reply:
Unfortunately it breaks calldata tests.